int64 IsManage
---
string FeedbackMsg
